wsd/DocumentBroker.cpp

```diff
@@ -2673,23 +2673,15 @@ bool DocumentBroker::sendUnoSave(const std::shared_ptr<ClientSession>& session,
         // We do not want save to terminate editing mode if we are in edit mode now.
         //TODO: Perhaps we want to terminate if forced by the user,
         // otherwise autosave doesn't terminate?
-        oss << "\"DontTerminateEdit\":"
-               "{"
-               "\"type\":\"boolean\","
-               "\"value\":true"
-               "}";
+        oss << "\"DontTerminateEdit\" : { \"type\":\"boolean\", \"value\":true\" }";
```

Comment:
no, I was close, but its a stray \", https://github.com/CollaboraOnline/online/pull/8776 for that at least